src/dbml.pegjs

```diff
@@ -18,19 +18,22 @@ Option = key:OptionKey _ ":" _ value:OptionValue _ Comment?
 OptionKey = Name
 OptionValue = AnyString
 
-Table = "Table"i _ name:TableName _ alias:TableAlias? _ "{" __ items:TableItems Comment? __ "}"
-  { return { type: "table", name, items: items || [], alias }}
+SchemaName = Name
+FullTableName = schema:(SchemaName ".")? name:TableName
```

Comment:
Could you also add a test containing a table with a schema name? If it could also exercise the reference rules, that would be great!

Reply:
See `advanced_dbml.dbml`

Reply:
Looks quite nice, thanks a lot!